src/DDTrace/Integrations/Curl/CurlIntegration.php

```diff
@@ -63,14 +63,48 @@ public static function load()
 
             $info = curl_getinfo($ch);
             $sanitizedUrl = Urls::sanitize($info['url']);
+            unset($info['url']);
+
             if ($globalConfig->isHttpClientSplitByDomain()) {
                 $span->setTag(Tag::SERVICE_NAME, Urls::hostnameForTag($sanitizedUrl));
             } else {
                 $span->setTag(Tag::SERVICE_NAME, 'curl');
             }
             $span->setTag(Tag::RESOURCE_NAME, $sanitizedUrl);
+
+
+            // Special case the Datadog Standard Attributes
+            //  https://docs.datadoghq.com/logs/processing/attributes_naming_convention/
+
             $span->setTag(Tag::HTTP_URL, $sanitizedUrl);
-            $span->setTag(Tag::HTTP_STATUS_CODE, $info['http_code']);
+
+            tagFromCurlInfo($span, $info, Tag::HTTP_STATUS_CODE, 'http_code');
+
+            // Datadog sets durations in nanoseconds - convert from seconds
+            $span->setTag('duration', $info['total_time'] * 1000000000);
+            unset($info['duration']);
+
+            tagFromCurlInfo($span, $info, 'network.client.ip', 'local_ip');
+            tagFromCurlInfo($span, $info, 'network.client.port', 'local_port');
+
+            tagFromCurlInfo($span, $info, 'network.destination.ip', 'primary_ip');
+            tagFromCurlInfo($span, $info, 'network.destination.port', 'primary_port');
+
+            tagFromCurlInfo($span, $info, 'network.bytes_read', 'size_download');
+            tagFromCurlInfo($span, $info, 'network.bytes_written', 'size_upload');
+
+
+            // Add the rest to a 'curl.' object
```

Comment:
Is the info available stable across the PHP versions we support? I'm a bit nervous about blindly adding this info.

Reply:
As Dan said below, it does seem pretty stable. My thinking was to not throw away anything that we're gathering, and that it would probably vary over time. 

A nice feature of the way we're calling `curl_getinfo` is that:

> Note that private data is not included in the associative array and must be retrieved individually with the CURLINFO_PRIVATE option.

from https://www.php.net/manual/en/function.curl-getinfo.php

So it should be safe to add whatever we get

Reply:
Any updates on this PR?